testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/ManagementInterfaceResourcesTestCase.java

```diff
@@ -159,13 +158,6 @@ private void runTest(boolean useSystemProperties, int noRequestTimeout, Exceptio
                 cli.sendLine(String.format(SYSTEM_PROPERTY_ADD, CONNECTION_LOW_WATER_PROPERTY, 3));
                 cli.sendLine(String.format(SYSTEM_PROPERTY_ADD, NO_REQUEST_TIMEOUT_PROPERTY, noRequestTimeout));
             } else {
-                cli.sendLine(String.format(HTTP_INTERFACE_READ_ATTRIBUTE, BACKLOG_ATTRIBUTE), true);
-                String response = cli.readOutput();
-                if (response.contains("WFLYCTL0201")) {
-                    LOG.info("Attribute \"backlog\" not found - assuming the attributes are not available at the server's stability level" );
-                    controller.stop();
-                    return;
-                }
```

Comment:
mmmh, I see this is going to fail at preview ... I'll fix it after CI initial check with the draft PR